src/Http/Http/src/IFileBufferingStreamFactory.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNetCore.WebUtilities;
+
+namespace Microsoft.AspNetCore.Http
+{
+    public interface IFileBufferingStreamFactory
```

Comment:
This would likely go in the Http.Abstractions package.

Reply:
I was thinking on putting `IBufferedWriteStream` on WebUtilities but in that case Http.Abstractions would have a dependency on it, is that OK?

Reply:
Hmm, tradeoffs. Let's not change the dependencies right now. That means IFileBufferingStreamFactory stays in Http and IBufferedWriteStream in WebUtilities.